test/e2e/framework/framework.go

```diff
@@ -55,6 +56,8 @@ type Framework struct {
 	Clientset_1_2 *release_1_2.Clientset
 	Clientset_1_3 *release_1_3.Clientset
 
+	FederationClientset *federation_internalclientset.Clientset
+	// TODO: remove FederationClient, all the client access must be through FederationClientset
```

Comment:
> @madhusudancs is the plan to remove FederationClient and switch over the ginkgo code to use FederationClientSet in a subsequent PR? 

@colhom yeah, that's the goal. 

> Is there anything blocking us from doing that now?

I don't want to change too many changes at once. First of all there is no guarantee that any of my stuff works because of the ongoing auth problems. Second, there is a time crunch at the moment and changing too many things mean a lot more debugging time.